db/schema.rb

```diff
@@ -139,6 +139,7 @@
     t.string "lh_source_plate_uuid", limit: 36, comment: "Source plate uuid created in crawler"
     t.datetime "created_at", comment: "When this record was inserted"
     t.datetime "updated_at", comment: "When this record was last updated"
+    t.index ["cog_uk_id"], name: "index_lighthouse_sample_on_cog_uk_id"
```

Comment:
Should it be unique? Can more than one sample row (e.g. if same root sample id tested more than once) have the same cog_uk_id.